amr-wind/core/SimTime.cpp

```diff
@@ -34,6 +39,38 @@ void SimTime::parse_parameters()
     } else {
         m_adaptive = true;
     }
+
+    if (m_plt_interval > 0 && m_plt_t_interval > 0.0) {
+        amrex::Abort(
+            "plot_interval and plot_time_interval are both specified. "
+            "timestep- and time-based plotting should not be used together; "
+            "please only specify one.");
+    }
+
+    if (m_chkpt_interval > 0 && m_chkpt_t_interval > 0.0) {
+        amrex::Abort(
+            "checkpoint_interval and checkpoint_time_interval are both "
+            "specified. timestep- and time-based checkpointing should not be "
+            "used together; please only specify one.");
+    }
```

Comment:
@ashesh2512 these two abort statements make sure that timestep- and time-based intervals are not being used together.